build/source/driver/multi_driver.f90

```diff
@@ -850,12 +851,14 @@ program multi_driver
    allocate(zSoilReverseSign(gru_struc(iGRU)%hruInfo(iHRU)%nSoil),stat=err); call handle_err(err,'problem allocating space for zSoilReverseSign')
    zSoilReverseSign(:) = -progStruct%gru(iGRU)%hru(iHRU)%var(iLookPROG%iLayerHeight)%dat(gru_struc(iGRU)%hruInfo(iHRU)%nSnow+1:nLayers)
   
+   nSoil = 9999
```

Comment:
No need for this line if defined as a parameter

Reply:
Line has been removed.